test/Microsoft.VisualStudio.SDK.Analyzers.Tests/VSSDK001DeriveFromAsyncPackageCodeFixTests.cs

```diff
@@ -256,43 +256,6 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
         await Verify.VerifyCodeFixAsync(test, withFix);
     }
 
-    [Fact]
-    public async Task InitializeOverride_AlreadyDefinesTaskUsing()
-    {
-        var test = @"
-using System;
-using Task = System.Threading.Tasks.Task;
-
-class Test : [|Microsoft.VisualStudio.Shell.Package|]
-{
-    protected override void Initialize()
-    {
-        base.Initialize();
-    }
-}
-";
-        var withFix = @"
-using System;
-using System.Threading;
-using Microsoft.VisualStudio.Shell;
-using Task = System.Threading.Tasks.Task;
-
-class Test : AsyncPackage
-{
-    protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
-    {
-        await base.InitializeAsync(cancellationToken, progress);
-
-        // When initialized asynchronously, we *may* be on a background thread at this point.
-        // Do any initialization that requires the UI thread after switching to the UI thread.
-        // Otherwise, remove the switch to the UI thread if you don't need it.
-        await this.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
-    }
-}
-";
-        await Verify.VerifyCodeFixAsync(test, withFix);
-    }
-
     [Fact]
```

Comment:
Why did this test cease to be interesting? Surely some user code exists with that code pattern. Don't we want to continue to verify that the code fix behaves properly in such code?

Reply:
Reverted